build.cake

```diff
@@ -27,7 +27,7 @@ var packageVersion = version + modifier + dbgSuffix;
 //////////////////////////////////////////////////////////////////////
 
 var WindowsFrameworks = new string[] {
-    "net-4.5", "net-4.0", "net-3.5", "net-2.0", "portable" };
+    "net-4.5", "net-4.0", "net-3.5", "net-2.0", "netstandard", "portable" };
 
```

Comment:
@ChrisMaddock just so people are not confused by your comment, the packaging of the ZIP is done, but I had to add `netstandard` there too. We should refactor it to use this list. 👍